src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -251,7 +251,8 @@
     <value>Launch size</value>
   </data>
   <data name="Globals_LaunchSize.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Defines whether the terminal will launch in a window, maximized, or full screen.</value>
+    <value>Defines whether the terminal will launch as maximized, full screen, or in a window. Setting this to "Focus" is equivalent to launching the terminal in the "Default" mode, but with the focus mode enabled. Similar, setting this to "MaximizedFocus" will result in launching the terminal in a maximized window with the focus mode enabled.</value>

```

Comment:
THE LOC FREEZE WAS THREE WEEKS AGO COME ON LOL

Reply:
also, the wording is wrong. `Maximized Focus` should be two words.

I don't think we're doing it right if we just copy the entire json schema field description into a freakin' tooltip, but it's what we've got today.

Reply:
My bad. #7873 went in and I had no clue. I'll pull all the resw changes into a separate PR to get enough time for localization.

Reply:
Moved all the localization stuff over to #8116. Removing this stuff from this PR.